AmplifyPlugins/DataStore/Sources/AWSDataStorePlugin/Sync/MutationSync/OutgoingMutationQueue/OutgoingMutationQueue.swift

```diff
@@ -55,7 +52,7 @@ final class OutgoingMutationQueue: OutgoingMutationQueueBehavior {
 
         let operationQueue = OperationQueue()
         operationQueue.name = "com.amazonaws.OutgoingMutationOperationQueue"
-        operationQueue.underlyingQueue = mutationDispatchQueue
+        operationQueue.qualityOfService = .default
```

Comment:
i think the default is `.default` so no need to set this

Reply:
According to [the doc](https://developer.apple.com/documentation/foundation/operationqueue/1417919-qualityofservice), the default value for the `OperationQueue` is `NSOperationQualityOfServiceBackground`. I set it to the `.default` because `DispatchQueue.global()` also uses a default quality of service (`qos = .default`).

>For queues you create yourself, the default value is NSOperationQualityOfServiceBackground. For the queue returned by the [main](https://developer.apple.com/documentation/foundation/operationqueue/1409193-main) method, the default value is NSOperationQualityOfServiceUserInteractive and cannot be changed.

Reply:
ah, got it, thanks